testing/docker.go

```diff
@@ -159,6 +166,32 @@ func (d *DockerContainer) Logs() (io.ReadCloser, error) {
 	})
 }
 
+func (d *DockerContainer) mappingForPort(cPort int) (containerPort uint, hostIP string, hostPort uint, err error) {
```

Comment:
Cockroach exposes two ports - the database protocol port (26257) and a web interface (8080).

This function needed to be added to allow for getting the mapped port that the database is bound to, otherwise the tests would attempt to connect to the UI.